includes/Fragments/RequestData.php

```diff
@@ -215,7 +215,8 @@ protected function setupRelatedRequests(
     {
         $this->assign('canSeeRelatedRequests', true);
 
-        $relatedEmailRequests = RequestSearchHelper::get($database)
+        // TODO: Do we want to return results from other domains?
```

Comment:
I'm personally leaning yes; it might be helpful information, but on the other hand there's probably a privacy policy angle here too, depending on how strictly the domains will be separated in terms of procedure and administration,

Reply:
Yeah, I'm thinking yes for now too, and that's what this code does.

I'm expecting us to have shared tool admins at least to begin with, and maybe some shared users until each domain is relatively self-sufficient. The code is written such that each domain *can* operate as an almost-independent instance of the tool (user status cross-domain is still something I need to figure out) and I expect/hope it will operate that way with a level of shared basic policy.